src/dscigametrics/stat_summary.py

```diff
@@ -0,0 +1,19 @@
+def stat_summary(metric, verbose = True):
+    """Return statistics summary of metrics
+
+    Parameters
```

Comment:
parameters should be campaign_id, start_date and end_date. and you should include parameters type (str, int, etc) and return items type. I think your function should return a table with four metrics and there mean, median, sd,...

Reply:
Edited, please check the modifications~